inst/tinytest/test_sparseConversion.R

```diff
@@ -690,44 +690,40 @@ if (suppressMessages(require(slam))) {
     }
 }
 
-## this fails persistently at CRAN on the Fedora box, but shouldn't
-## giving up and commenting out
-## https://www.r-project.org/nosvn/R.check/r-devel-linux-x86_64-fedora-clang/RcppArmadillo-00check.html
 ## test.stop <- function() {
-##     ## [Matrix] p87 (lgCMatrix)
-##     m <- Matrix(c(0,0,2:0), 3,5, dimnames=list(LETTERS[1:3],NULL))
-##     lm <- (m > 1)
-##     checkException(asSpMat(lm))
+## [Matrix] p87 (lgCMatrix)
+m <- Matrix(c(0,0,2:0), 3,5, dimnames=list(LETTERS[1:3],NULL))
+lm <- (m > 1)
+expect_error(asSpMat(lm))
```

Comment:
Careful. If memory serves I went through several rounds of uploads and test "tweaking" yet it still failed at the sometimes very idiosyncratic setup of these boxen by BDR.

I am not in favour of turning these tests back on.  AFAICT this is not an issue about testing exceptions or errors between `RUnit` and `tinytest`.

Reply:
I see. Then let me close this PR.